cmd/modules/analyzers/test-analyzer/main.go

```diff
@@ -154,19 +154,24 @@ func TestPackageNode(t *testing.T) {
 }
 
 func TestCurlBuildGraph(t *testing.T) {
-	if pkgNode.Targets[0].Path != "/buildroot/curl/build/src/curl" && pkgNode.Targets[1].Path != "/buildroot/curl/build/src/curl" {
+	if len(pkgNode.Targets) < 2 {
+		t.Logf("Package node '%v' is not connected to all the configured linked targets", pkgNode.Name)
+		t.Fail()
+	} else if pkgNode.Targets[0].Path != "/buildroot/curl/build/src/curl" && pkgNode.Targets[1].Path != "/buildroot/curl/build/src/curl" {
 		t.Logf("Package node %v is not connected to curl linked target", pkgNode.Name)
 		t.Fail()
-	}
-	if pkgNode.Targets[0].Path != "/buildroot/curl/build/lib/libcurl.so" && pkgNode.Targets[1].Path != "/buildroot/curl/build/lib/libcurl.so" {
+	} else if pkgNode.Targets[0].Path != "/buildroot/curl/build/lib/libcurl.so" && pkgNode.Targets[1].Path != "/buildroot/curl/build/lib/libcurl.so" {
```

Comment:
Please don't use else if, it is ugly and becomes hard to maintain. Since the actual body is also duplicated use a loop.